Neos.Flow/Classes/Persistence/Doctrine/Query.php

```diff
@@ -12,15 +12,18 @@
  */
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception;
```

Comment:
I usually do
```suggestion
use Doctrine\DBAL\Exception as DBALException;
```

To make the code a bit more explicit – and in this case it would make the diff slightly smaller